hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java

```diff
@@ -609,6 +615,64 @@ private void handleUserTransaction(
         handleWorkflowMetrics.update(transactionInfo.functionality(), handleDuration);
     }
 
+    /**
```

Comment:
Deja vu.  Were these change not also [here](https://github.com/hashgraph/hedera-services/pull/11823/files)?

Reply:
For `CryptoTransfer`'s with zero-adjustments, yes; this includes further cases.